include/uwrt_mars_rover_utils/hw_bridge.h

```diff
@@ -1,14 +1,21 @@
 #pragma once
 #include <string>
 
+#include "CANMsgMap.h"
+#include "uwrt_mars_rover_can.h"
+#include "uwrt_mars_rover_can_enums.h"
+#include "uwrt_mars_rover_can_wrapper.h"
+
 namespace HWBRIDGE {
+
 namespace ROVERCONFIG {
 constexpr uint16_t ROVER_MOTOR_PWM_FREQ_HZ = 20000;   // 20 kHz
 constexpr uint32_t ROVER_CANBUS_FREQUENCY  = 500000;  // 500 kbps
```

Comment:
forgot to change this

Reply:
actually a lot of this section doesnt even belong in the bridge i dont think. some things vars only concern fw